src/components/globals/Header/Header.scss

```diff
@@ -0,0 +1,104 @@
+.header {
+  position: sticky;
+  top: 0;
+  border-radius: 0 0 16px 16px;
+  padding: 12px 0;
+  height: 64px;
+  box-shadow: 0 2px 16px 0 $color-black-10;
+  background: $color-white;
+
+  &__wrapper {
+    justify-content: space-between;
+  }
+
+  &__wrapper,
+  &__right,
+  &__left,
+  &__buttons {
+    display: flex;
+    align-items: center;
+  }
+
+  @include desktop {
+    border-radius: 0 0 32px 32px;
+    padding: 24px 0;
+    height: 97px;
+
+    &__contact-button {
+      margin-right: 16px;
+      padding: 12px 24px;
+      width: 141px;
```

Comment:
I think we shouldn't restrict width of a button, especially if it's going to be manageable in CMS. Maybe we can use min-width if we need, but let the button stretch if there's more text. Right now it would look like this if there's more text: 
<img width="327" alt="Screenshot 2024-11-04 at 7 11 08 PM" src="https://github.com/user-attachments/assets/f549e115-f87f-4a2d-8146-28f1a18b0027">